datasets/slides/boston.rmd

```diff
@@ -0,0 +1,144 @@
+---
```

Comment:
Please include the discussion of potential ethical problems for the Boston housing dataset, see, e.g., https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html